src/components/custom-dashboard/card-switch.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { setCardType } from "../cards";
+import { Layout } from "react-grid-layout";
+
+export function cardSwitch<T, CO>(layoutBindings: LayoutBindings<T, CO>) {
+  return ({ i }: Layout) => {
+    const Card = setCardType(layoutBindings[i].cardType);
```

Comment:
Is it ok to do this set thing on every render, re-creating the Card component which will be rendered (leading to unmounting of further content in the tree)? Or something takes care of it?

Reply:
I want to make it under the `memo`. In future, to prevent rerenders.